client-service-worker/src/__dumber-gist-worker.js

```diff
@@ -132,7 +134,7 @@ addEventListener('fetch', e => {
       return fetch(e.request).then(response => {
         if (!response.ok && pathname.length > 1) {
           // Try font/image from jsdelivr
-          return fetch(`//cdn.jsdelivr.net/npm${pathname}`, {mode: 'cors'}).then(response => {
+          return fetch(`${JSDELIVR_CDN_URL}/npm${pathname}`, {mode: 'cors'}).then(response => {
```

Comment:
Since you swap jsdelivr to some private cdn, I guess it only works with those share same restful api as jsdelvr.
Namely a private clone of jsdelivr.

Reply:
Yes, in the current idea it can only work with a clone of jsdelivr (or via a compatibility layer).